src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.Paint.cs

```diff
@@ -19,12 +23,28 @@
 using SkiaSharp.Views.UWP;
 #endif
 
+#if __ANDROID__ && ANDROID_REFERENTIAL_IMPL
+using _SKXamlCanvas = SkiaSharp.Views.Windows.SKSwapChainPanel;
+using _SKPaintSurfaceEventArgs = SkiaSharp.Views.Windows.SKPaintGLSurfaceEventArgs;
+#else
+using _SKXamlCanvas = SkiaSharp.Views.Windows.SKXamlCanvas;
+using _SKPaintSurfaceEventArgs = SkiaSharp.Views.Windows.SKPaintSurfaceEventArgs;
```

Comment:
add a permutation for `SkiaSharp.Views.Windows.*` aliases where `Windows` can be: Windows or UWP (based on `#if IS_WINUI`)

Reply:
nvm... there is only winui build for this